codegen-renderers/src/test/resources/java/ramlTestFiles/generated/SimpleType.txt

```diff
@@ -0,0 +1,186 @@
+package com.commercetools.test.models.simpleTypes;
```

Comment:
cool this would allow us to see the diff, before validating new generator 👍 
I really like it 💯